property_test.go

```diff
@@ -107,6 +110,26 @@ func TestProperty(t *testing.T) {
 		require.False(t, ok)
 		require.Nil(t, property1)
 	})
+
+	t.Run("Info", func(t *testing.T) {
+		err := testTypeSilent.New("test").WithProperty(testInfoProperty2, "hello world")
+		assert.Equal(t, "foo.bar.silent: test {prop2: hello world}", err.Error())
+		// overwrite
+		err0 := err.WithProperty(testInfoProperty2, "cruel world")
+		assert.Equal(t, "foo.bar.silent: test {prop2: cruel world}", err0.Error())
+		// add more
+		err1 := err.WithProperty(testInfoProperty3, struct{ a int }{1})
+		assert.Equal(t, "foo.bar.silent: test {prop3: {1}, prop2: hello world}", err1.Error())
+		// decorate
+		err2 := Decorate(err, "oops")
+		assert.Equal(t, "oops, cause: foo.bar.silent: test {prop2: hello world}", err2.Error())
+		// decorate and add more
+		err3 := err2.WithProperty(testInfoProperty3, struct{ a int }{1})
+		assert.Equal(t, "oops, cause: foo.bar.silent: test {prop2: hello world} {prop3: {1}}", err3.Error())
```

Comment:
Here it seems like an issue: testInfoProperty3 is added to wrapped error `err2`, but message looks like it is the property of original `err`. 
It seems to look better this way, but it also creates a confusion which seems dangerous. First and foremost, the error message should be clear to decypher.

Reply:
In case of more than three errors wrapping each other it'll be difficult to say whom the printable property belongs to. We can wrap all causes with parentheses, but it'll make all error messages uglier.

Reply:
Here's what kind of shape I am talking about:
`wrap3message {wrap3property: foo}, cause: wrap2message {wrap2property: bar}, cause: originalError (hidden: underlying errors)`